lib/SPIRVProducerPass.cpp

```diff
@@ -6777,6 +6777,7 @@ void SPIRVProducerPassImpl::GenerateKernelReflection() {
     // Ops[1] = reflection ext import
     // Ops[2] = function id
     // Ops[3] = kernel name
+    // Ops[4] = number of arguments
```

Comment:
Ack. This is an optional argument in rev5 of NonSemantic.ClspvReflection

https://github.com/KhronosGroup/SPIRV-Headers/blob/main/include/spirv/unified1/extinst.nonsemantic.clspvreflection.grammar.json#L10